src/locales/en_US/en_US.json

```diff
@@ -16,11 +16,18 @@
             "last_name": "Last name",
```

Comment:
could you please add strings for all languages at the same time please ?

Reply:
I don't have access to lokalise and I don't have the all the strings. I already asked for them, so I'll add them as soon as I can.

Reply:
there's a unit test in the dev branch that will fail if the locales are not in sync with the en_US file, just fyi :)